tests/app/main/test_frameworks.py

```diff
@@ -465,9 +471,16 @@ def _assert_email(self, send_email, is_called=True):
                 "G-Cloud 7 Supplier",
                 ["clarification-question"]
             )
-
-        else:
-            assert_equal(0, send_email.call_count)
+        if succeeds:
+            send_email.assert_any_call(
+                "email@email.com",
+                mock.ANY,
+                "MANDRILL",
+                "Thanks for your clarification question",
+                "do-not-reply@digitalmarketplace.service.gov.uk",
+                "Digital Marketplace Admin",
+                ["clarification-question-confirm"]
+            )
```

Comment:
Yeah, it's nice the logic still (mostly) holds together without your having to do anything too crazy.